repo.go

```diff
@@ -605,7 +607,10 @@ func (r *Repo) getSigningKeys(name string) ([]sign.Signer, error) {
 		return nil, err
 	}
 	if name == "root" {
-		return signingKeys, nil
+		sorted := make([]sign.Signer, len(signingKeys))
+		copy(sorted, signingKeys)
+		sort.Sort(signer.ByIDs(sorted))
```

Comment:
This line is a bit hard to parse. I am sort sorting signer which are already sorted?

Reply:
Lines 610-611 makes a copy of `signingKeys` called `sorted` (which is not yet sorted until the next line). Line 612 sorts `sorted` in place using the `signer.ByIDs` comparator.